src/pages/[platform]/build-a-backend/auth/manage-attributes/index.mdx

```diff
@@ -0,0 +1,645 @@
+        case AuthUpdateAttributeStep.confirmAttributeWithCode:
+          final destination = value.nextStep.codeDeliveryDetails?.destination;
+          safePrint('Confirmation code sent to $destination for $key');
+          break;
+        case AuthUpdateAttributeStep.done:
+          safePrint('Update completed for $key');
+          break;
+      }
+    });
+  } on AuthException catch (e) {
+    safePrint('Error updating user attributes: ${e.message}');
+  }
+}
+```
+
+## Verify user attribute
+
+Some attributes require confirmation for the attribute update to complete. If the attribute need to be confirmed, the result of the above API will be `CONFIRM_ATTRIBUTE_WITH_CODE`. A confirmation code will be sent to the delivery medium mentioned in the delivery details. When the user gets the confirmation code, you can present a UI to the user to enter the code and invoke the confirm attribute API with their input:
+
+```dart
+Future<void> verifyAttributeUpdate() async {
+  try {
+    await Amplify.Auth.confirmUserAttribute(
+      userAttributeKey: AuthUserAttributeKey.email,
+      confirmationCode: '390739',
+    );
+  } on AuthException catch (e) {
+    safePrint('Error confirming attribute update: ${e.message}');
+  }
+}
+```
+
+## Resend verification code
+
+If the code has expired or the user needs to resend the confirmation code, invoke the resend API as shown below:
+
+ ```dart
+Future<void> resendVerificationCode() async {
+  try {
+    final result = await Amplify.Auth.resendUserAttributeConfirmationCode(
+      userAttributeKey: AuthUserAttributeKey.email,
+    );
+    _handleCodeDelivery(result.codeDeliveryDetails);
+  } on AuthException catch (e) {
+    safePrint('Error resending code: ${e.message}');
+  }
+}
+```
+
+## Custom attributes
+Amplify Flutter supports [standard OIDC user attributes](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) as well as custom attributes. Custom attributes can be instantiated via the custom attribute constructor:
+
+```dart
+Future<void> _signUp({
+    required String username,
+    required String password,
+    required String email,
+    required String customValue,
+}) async  {
+  final userAttributes = {
+    AuthUserAttributeKey.email: email,
+    // Create and pass a custom attribute
+    const CognitoUserAttributeKey.custom('my-custom-attribute'): customValue
+  };
+  await Amplify.Auth.signUp(
+    username: username,
+    password: password,
+    options: SignUpOptions(
+      userAttributes: userAttributes,
+    ),
+  );
+}
+```
+
+When working with a Cognito UserPool, you can set up [custom attributes](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#user-pool-settings-custom-attributes) via the Cognito console or AWS CLI. Although Cognito prepends a "custom:" prefix on the attribute name, there is no need for you to add this in Amplify Flutter's custom attribute constructor.
```

Comment:
can we link to the custom attributes example in the auth docs? https://next-docs.amplify.aws/react/build-a-backend/auth/override-cognito/#custom-attributes